tutorials/users/remote-processing/remote-processing.md

```diff
@@ -1,7 +1,199 @@
 # Remote Processing
 
-```{important}
-This tutorial is coming soon!
+Remote processing services allow users to run workflows on different nodes across the Marble network.
 
-Check back soon for more content.
+By distributing climate data across the nodes in the Marble network, users can choose to run their workflows on the same
+servers as the data. In other words, instead of downloading large amounts of data in order to run workflows on their own
+computers, users can send their workflows to the computer that hosts the data to run there. This reduces lengthy
+download times and eliminates the need to duplicate data unnecessarily.
+
+The [Weaver](https://pavics-weaver.readthedocs.io) service provides remote processing capabilities across the Marble 
+Network. Any node that enables the Weaver service will allow users to run workflows on that node from anywhere else in 
+the network.
+
+## How Weaver works
+
+Weaver is an execution management service (EMS) where users can send instructions in the form of workflows which are 
+then executed by different processes across the network. A workflow can contain a single process step or multiple.
+
+Processes can either be run in synchronous or asynchronous mode. 
+
+- A synchronous process will block until it has finished and then return the result of the process. 
+
+- An asynchronous process will immediately return a response containing a URL that can be used to check the status of 
+  the process and to retrieve the result once the process has finished. A user running an asynchronous process is 
+  responsible for checking back later to check if the process has finished.
+
+```{note}
+If a synchronous process takes too long (reaches a timeout threshold), the process will be converted to asynchronous
+mode. 
+```
+
+### Atomic Processes
+
+Weaver supports several types of atomic processes. These processes receive inputs in the form of data and customization
+arguments and returns the result of that process to the user.
+
+Atomic processes include:
+
+- [builtin](https://pavics-weaver.readthedocs.io/en/latest/processes.html#proc-builtin)
+  - These processes come built into Weaver. No additional steps are required to deploy these processes.
+- [Web Processing Service (WPS)](https://pavics-weaver.readthedocs.io/en/latest/processes.html#wps-1-2)
+  - These processes are executed by a remote server through the [WPS 1/2](https://www.ogc.org/standard/wps/) interface.
```

Comment:
Should the Web Processing Service and WPS 1/2 links be reversed?

Reply:
No, one is a link to the weaver documentation, the other is to the OGC standard definition

Reply:
Ok.